src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -457,27 +448,27 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs args)
                 switch (args.Kind)
                 {
                     case WorkspaceChangeKind.ProjectChanged:
+                    {
```

Comment:
I am not sure. why blocking { for case statement? did we ever do?

Reply:
@dotnet/roslyn-ide is there a reason why we chose our format standard to indent { in case statement? I dont think we ever did that in roslyn code base? why did we change them all?